src/actions/ActionTypes.js

```diff
@@ -4,6 +4,7 @@ export const CONNECT_SUCCESSFUL = "CONNECT_SUCCESSFUL";
 export const CONNECT_FAILED = "CONNECT_FAILED";
 export const LOGIN_SUCCESSFUL = "LOGIN_SUCCESSFUL";
 export const LOGIN_FAILED = "LOGIN_FAILED";
-export const SOMETHING_HAPPENED_SUCCESSFULLY = "SOMETHING_HAPPENED_SUCCESSFULLY";
+export const SOMETHING_HAPPENED_SUCCESSFULLY =
```

Comment:
Please do not enforce such a short line width.

Reply:
Fixed.